queue/remote_base.go

```diff
@@ -293,6 +294,13 @@ func (q *remoteBase) CompleteRetrying(ctx context.Context, j amboy.RetryableJob)
 					j.AddError(catcher.Resolve())
 					return errors.Wrapf(catcher.Resolve(), "giving up after attempt %d", attempt)
 				}
+				grip.Debug(message.WrapError(err, message.Fields{
+					"message":  "failed to mark job as completed retrying",
```

Comment:
Should this say `failed to mark retrying job as completed`?

Reply:
Your phrasing sounds way better - done.